GuiTest/src/main/java/io/github/cottonmc/test/client/PaddingTestGui.java

```diff
@@ -0,0 +1,35 @@
+package io.github.cottonmc.test.client;
+
+import io.github.cottonmc.cotton.gui.client.LightweightGuiDescription;
+import io.github.cottonmc.cotton.gui.widget.WGridPanel;
+import io.github.cottonmc.cotton.gui.widget.WLabel;
+import io.github.cottonmc.cotton.gui.widget.data.HorizontalAlignment;
+import io.github.cottonmc.cotton.gui.widget.data.Insets;
+import io.github.cottonmc.cotton.gui.widget.data.VerticalAlignment;
+
+import net.minecraft.text.Text;
+
+public class PaddingTestGui extends LightweightGuiDescription {
+
```

Comment:
There should be no empty lines like this in the start/end of the class, please stick to the existing code style

Also constructors should come first in the class